src/main/java/com/matera/account/account/Account.java

```diff
@@ -0,0 +1,108 @@
+package com.matera.account.account;
+
+import java.math.BigDecimal;
+import java.util.UUID;
+import javax.persistence.*;
+
+@Entity
+public class Account implements AccountProjection{
+
+    // Constructor's
+
+    public Account(){}
+
+    public Account(UUID clientId, AccountDTO accountDTO) {
+        this.accountId = accountDTO.getAccountId();
+        this.clientId = clientId;
+        this.accountTypeId = accountDTO.getAccountTypeId();
+        this.agency = accountDTO.getAgency();
+        this.accountNumber = accountDTO.getAccountNumber();
+        this.balance = accountDTO.getBalance();
+    }
+
+
+    // Properties
+
+    @Id
+    @Column(name = "ACCOUNT_ID")
```

Comment:
@Column annotations are redundant here because spring data transaltes "account_id" database column to accountId java attribute automatically by convention